responses/tests/test_responses.py

```diff
@@ -1058,7 +1094,7 @@ def run():
             body="ok",
             stream=responses_stream,
         )
-        session = requests.session()
+        session = requests.Session()
```

Comment:
the `session` with `s` lowercased is actually deprecated since a long time in Requests (since v1 if I recall correctly).
and removed in Niquests.